plugins/backstage-plugin-env0/src/components/env0-variables-input/env0-variable-field.tsx

```diff
@@ -12,7 +12,7 @@ import type { Variable } from '../../api/types';
 
 const VariableContainer = styled('div')(() => ({
   display: 'grid',
-  gridTemplateColumns: '1fr 5fr',
+  gridTemplateColumns: '1.5fr 5.5fr',
```

Comment:
I think natural numbers would look better, so 3 11 (also we should probably see that it looks ok with the required triangle and the variable description icon)

Reply:
8a4d5e76(natural numbers)